packages/dd-trace/src/appsec/iast/taint-tracking/operations.js

```diff
@@ -0,0 +1,81 @@
+const { storage } = require('../../../../../datadog-core')
+const iastContextFunctions = require('../iast-context')
+const log = require('../../../log')
+const TaintedUtils = require('@datadog/native-iast-taint-tracking')
+
+const IAST_TRANSACTION_ID = Symbol('_dd.iast.transactionId')
+
+function noop (res) { return res }
+const TaintTrackingDummy = {
+  plusOperator: noop
+}
+
+const TaintTracking = {
+  plusOperator: function (res, op1, op2) {
+    try {
+      if (typeof res !== 'string' ||
+        (typeof op1 !== 'string' && typeof op2 !== 'string')) { return res }
+
+      const store = storage.getStore()
+      const iastContext = iastContextFunctions.getIastContext(store)
+      const transactionId = iastContext && iastContext[IAST_TRANSACTION_ID]
+      if (transactionId) {
+        res = TaintedUtils.concat(transactionId, res, op1, op2)
+      }
+    } catch (e) {
+      log.debug(e)
+    }
+    return res
+  }
+}
+
+class TaintOperations {
```

Comment:
Why use a class instead of "normal" object like `TaintTracking`?
We have to do a `new TaintOperations` that I think that isn't necessary